sanic/sanic.py

```diff
@@ -1,5 +1,9 @@
-import asyncio
+from argparse import ArgumentParser
+from asyncio import get_event_loop
 from inspect import isawaitable
+from multiprocessing import Process, Event
```

Comment:
If run is executed with 1 worker, it won't use multiprocessing.  Should we not import it by default?